message-signing/js/ps-signing/package.json

```
@@ -0,0 +1,14 @@
+{
+  "name": "ps-signing",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "Saffaanh Soobratty <saffaanh.soobratty@starlingbank.com>",
+  "license": "ISC",
```

```suggestion
  "license": "MIT",
```

(or just remove this license line - there's a copy of the MIT license in the repository root already :))